lambda/test_action_network.py

```diff
@@ -0,0 +1,154 @@
+import pprint
+
+from action_network import (convert_event, get_email_address_from_organizer,
+                            get_event_name, get_events_from_events_campaign,
+                            get_object_or_empty_dict, get_organizer,
+                            make_location)
+from action_network_sample_data import make_test_event, make_test_location
+
+
+def test_make_location_everything():
+    location = make_location({'location': make_test_location()})
+    assert location == '10025'
+
+
+def test_make_location_nothing():
+    inner_loc = make_test_location()
+    del inner_loc['locality']
+    del inner_loc['region']
+    del inner_loc['postal_code']
+    location = make_location({'location': inner_loc})
+    assert location == ''
+
+
+def test_make_location_none():
+    location = make_location({'location': None})
+    assert location == ''
+
+
+def test_make_location_not_us():
+    inner_loc = make_test_location()
+    inner_loc['country'] = 'New Zealand'
+    location = make_location({'location': inner_loc})
+    assert location == 'New York, New Zealand'
+
+
+def test_get_object_or_empty_dict_empty_dict():
+    x = get_object_or_empty_dict('foo', {})
```

Comment:
no need to write tests about python builtins

Reply:
get_object_or_empty_dict was not a builtin

and as I said above, I feel more confident seeing an explicit test than I do relying on my memory of the intricacies of the behavior of object.get